src/modules/iotjs_module_adc.h

```diff
@@ -34,7 +34,8 @@ typedef struct {
 
 #if defined(__linux__)
   iotjs_string_t device;
-#elif defined(__NUTTX__)
+#endif
+#if defined(__NUTTX__) || defined(__TIZENRT__)
```

Comment:
It is only a minor style issue. BTW, you wrote the `#elif` style in `src/modules/iotjs_module_adc.c`. I just made a quick search in the code base and we are using the same style for this in the whole code base. Please don't break it.